client/app/services/dashboard.js

```diff
@@ -2,6 +2,8 @@ import _ from 'lodash';
 
 export let Dashboard = null; // eslint-disable-line import/no-mutable-exports
 
+export const IS_DASHBOARD_PARAM_SOURCE = Symbol('dash_source');
```

Comment:
Someone should definitely check how it works in IE: IE does not support symbols at all, and due to it's nature symbols cannot be polyfilled.

Reply:
Swapped for normal string